tests/TestCase/Console/Command/HelpCommandTest.php

```diff
@@ -38,10 +39,13 @@ public function setUp(): void
         $this->setAppNamespace();
         Plugin::getCollection()->clear();
 
-        $app = $this->getMockForAbstractClass(
```

Comment:
This is somewhat easier to read from testing perspective
I wonder if we should just keep our own version instead of trying to work with harder to read and maintain anonymous classes?

Reply:
I personally prefer the anonmyous classes only if there is very little to implement.
Also with this we are not dependent on functionality which may or may not be removed in the future.